### ISSUE TITLE
[Accessibility]A11y_.NET Online REPL View_Color Contrast: The color contrast ratio for the line numbers in the editor is 3.7:1 which is < 4.5:1 #105

### ISSUE THREAD

**Issue by LadyNaggaga** [Microsoft/monaco-editor#1164]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/105) . I have also included the details below 

> **Environment Details:** 
> URL: https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
> Browser: Edge 42.17134.1.0
> 
> **Repro Steps:**
> 1)Hit the URL : https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
> 2)Tab till "Enter focus mode" button and press enter
> 3)Check the color contrast ratio for all the contents in the editor pane
> 
> **Actual Result:**
> The color contrast ratio for the line numbers in the editor is 3.7:1 which is < 4.5:1
> 
> **Expected Result:**
> The color contrast ratio for the line numbers in the editor should be >= 4.5:1
> 
> **User Impact:**
> Visually impaired users will find it difficult to access the line numbers when errors are shown in the output section using line number
> 
> **MAS Reference:**
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D)
> ![bug_color contrast for numbers](https://user-images.githubusercontent.com/41108985/42629429-e45831fe-85f0-11e8-9f13-28c94fab120d.png)

**Comment by alexandrudima** [comment]
We are shipping with a High Contrast Theme

Please see https://github.com/Microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators
